### DISCUSSION TITLE
Data sources used for ogbl-biokg

### DISCUSSION THREAD

**Discussion by sophiakrix** [snap-stanford/ogb#278] [General]
Hi there,

I was wondering whether I can find any information about which databases were used to construct the ogbl-biokg. Especially, it would be good to know where the edges come from and if they have any evidence score, such as from genetic association or literature search.

Any info would be appreciated!

Best,

Sophia

**Comment by sophiakrix** [comment]
Hi there, 

I was wondering if I could get any further information on this point? That would be very helpful.

Best,

Sophia

**Reply by weihua916** [reply]
see https://github.com/snap-stanford/ogb/issues/111

**Reply by sophiakrix** [reply]
Thanks, I wrote an email. Hope to get more insight on this soon!

**Reply by sophiakrix** [reply]
Hi @weihua916 !
Hope you're doing well. I guess you've seen the email I sent to Marinka with you in CC. Since I haven't gotten a response yet, I wanted to ask if you could ask Marinka again about this issue. Me and others in the field would very much appreciate this.
Best,
Sophia
